rust/Cargo.toml

```diff
@@ -39,6 +41,11 @@ arboard = { version = "3.4.0", default-features = false }
 async-trait = { version = "0.1", default-features = false }
 atomicwrites = "0.4.4"
 axum = { version = "0.7.7", default-features = false }
+aya = { git = "https://github.com/aya-rs/aya" }
+aya-build = { git = "https://github.com/thomaseizinger/aya", branch = "fix/aya-build-pin-nightly" }
```

Comment:
Depending on a fork here until https://github.com/aya-rs/aya/pull/1228 is merged.